File: Assets/Bundles/Server/Resources/TS/Services/Global/Chat/Commands/ChatCommand.lua
-- Compiled with unity-ts v2.1.0-75
local ChatCommand
do
	ChatCommand = {}
	function ChatCommand:constructor(commandLabel, aliases)
		if aliases == nil then
			aliases = {}
		end
		self.commandLabel = commandLabel
		self.aliases = aliases
	end
end
return {
	ChatCommand = ChatCommand,
}
-- ----------------------------------
-- ----------------------------------

File: Assets/Bundles/Imports/Core/Core~/src/Shared/Commands/ChatCommand.ts
import { Player } from "Shared/Player/Player";

export abstract class ChatCommand {
	constructor(
		public commandLabel: string,
		public aliases: string[] = [],
		public usage: string = "",
		public description: string = "",
	) {}

	public abstract Execute(player: Player, args: string[]): void;
}
